src/DBFTPlugin/Consensus/ConsensusContext.cs

```diff
@@ -264,6 +264,11 @@ public void Save()
             store.PutSync(ConsensusStateKey, this.ToArray());
         }
 
+        public void Delete()
+        {
+            store.Delete(ConsensusStateKey);
```

Comment:
We don't need to delete it. Because it will be replaced in the next block.

Reply:
Updated. How do you think?

Reply:
> We don't need to delete it. Because it will be replaced in the next block.

But will show error in log.
![1655953861544](https://user-images.githubusercontent.com/38493437/175199122-ca43752e-fbc7-4ad1-812f-2b2c42bd300c.png)

Reply:
This log is necessary. You use a wrong Neo.dll.